doc/changelog/04-tactics/14657-simple_congruence.rst

```diff
@@ -0,0 +1,8 @@
+- **Added:**
+  :tacn:`simple congruence` tactic which is essentially :tacn:`congruence`
+  but does not unfold definitions.
```

Comment:
```suggestion
  :tacn:`simple congruence` tactic which works like :tacn:`congruence`
  but does not unfold definitions.
```

If this is correct.  "essentially" seems more vague than necessary.

Reply:
Will change.